srcalc/src/main/java/gov/va/med/srcalc/web/controller/admin/NewRuleController.java

```diff
@@ -0,0 +1,136 @@
+package gov.va.med.srcalc.web.controller.admin;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+import javax.inject.Inject;
+
+import gov.va.med.srcalc.domain.model.Variable;
+import gov.va.med.srcalc.service.AdminService;
+import gov.va.med.srcalc.service.DuplicateRuleNameException;
+import gov.va.med.srcalc.service.InvalidIdentifierException;
+import gov.va.med.srcalc.util.ValidationCodes;
+import gov.va.med.srcalc.web.view.VariableSummary;
+import gov.va.med.srcalc.web.view.Views;
+import gov.va.med.srcalc.web.view.admin.EditRule;
+import gov.va.med.srcalc.web.view.admin.ValueMatcherBuilder;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Controller;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.servlet.ModelAndView;
+
+import com.google.common.primitives.Ints;
+
+/**
+ * <p>Web MVC controller for creating a new rule.</p>
+ */
+@Controller
+public class NewRuleController
+{
+    public static final String NEW_RULE_URL = "/admin/newRule";
```

Comment:
Everywhere else we call this constant `BASE_URL`. Any reason why this is different?